Dockerfile

```diff
@@ -31,11 +31,10 @@ RUN yum update -y \
   && cd /tmp/stack \
   && wget --output-document stack.tgz --no-verbose "https://github.com/commercialhaskell/stack/releases/download/v$STACK_VERSION/stack-$STACK_VERSION-linux-x86_64.tar.gz" \
   && tar --extract --file stack.tgz --strip-components 1 --wildcards '*/stack' \
-  && rm stack.tgz \
```

Comment:
No need to remove this file individually since we later delete the entire directory.